File: src/main/java/net/igneo/icv/entity/chestplate/soulSpider/SoulSpiderRenderer.java
package net.igneo.icv.entity.chestplate.soulSpider;

import net.minecraft.client.renderer.entity.EntityRendererProvider;
import software.bernie.geckolib.renderer.GeoEntityRenderer;

public class SoulSpiderRenderer extends GeoEntityRenderer<SoulSpiderEntity> {
    
    public SoulSpiderRenderer(EntityRendererProvider.Context renderManager) {
        super(renderManager, new SoulSpiderModel());
    }
}

File: src/main/java/net/igneo/icv/entity/chestplate/soulSpider/SoulSpiderRenderer.kt
package net.igneo.icv.entity.chestplate.soulSpider

import net.minecraft.client.renderer.entity.EntityRendererProvider
import software.bernie.geckolib.renderer.GeoEntityRenderer

class SoulSpiderRenderer(renderManager: EntityRendererProvider.Context) :
    GeoEntityRenderer<SoulSpiderEntity?>(renderManager, SoulSpiderModel())
